index.js

```diff
@@ -24,6 +26,26 @@ sysLogLevels = {
   },
 };
 
+/**
+ * setups up formatters for winston 3 format (https://github.com/winstonjs/winston#formats)
+ * @param {*} formatter options 
+ * @returns 
+ */
+function setupConsoleFormatters(options) {
+  var formatters = [];
+
+  if(options.json){
+    formatters.push(json());
+  }
+
+  if(options.prettyPrint || options.stringify == null){
+    let colorize = options.colorize == true ? true : false;
```

Comment:
NBD, but I think this can be simplified to

```suggestion
    const colorize = !!options.colorize;
```